pkg/route/ingress/ingress.go

```diff
@@ -882,3 +889,22 @@ func terminationPolicyForIngress(ingress *networkingv1.Ingress) routev1.TLSTermi
 		return routev1.TLSTerminationEdge
 	}
 }
+
+var destinationCACertificateSecretKeyName = "destinationCACertificate"
+var destinationCACertificateAnnotationKey = routev1.GroupName + "/destinationCACertificateSecret"
```

Comment:
Existing route and ingress annotation keys use dotted.case, snake_case, or kebab-case.  I'd prefer we not add camelCase to the menagerie.  I suggest using kebab-case because it appears to be the predominant style, and it is the style we have used in most recently added annotation keys for routes.  
```suggestion
var destinationCACertificateAnnotationKey = routev1.GroupName + "/destination-ca-certificate-secret"
```

Reply:
Changed the kebab-case